buildSrc/src/main/resources/pom.xml

```diff
@@ -145,5 +145,12 @@
             <version>4.1.1</version>
         </dependency>
 
+        <!-- Swagger Annotations -->
+        <dependency>
+            <groupId>io.swagger</groupId>
```

Comment:
When you add the open api feature in Micronaut CLI or Micronaut Launch we add the following depenency: 

https://github.com/micronaut-projects/micronaut-starter/blob/3.3.x/starter-core/src/main/java/io/micronaut/starter/feature/other/OpenApi.java#L56

`io.swagger.core.v3:swagger-annotations`

Here you are using `io.swagger:swagger-annotations` instead. Which one should we use in the guide and in starter?

Reply:
Can you update the guide to use `io.swagger.core.v3:swagger-annotations` in the meanwhile?

Reply:
We cannot change to `io.swagger.core.v3:swagger-annotations` now.
They have a lot of differences with the current version, and most imports do not work, e.g.:
`import io.swagger.annotations.ApiModel` -> `import io.swagger.v3.oas.annotations.media.Schema`.

This will have to be changed in the generator. After that we can update the guide.